### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by suyashmahar**
This PR refactors code to:
* Remove redundant tests from project. This stops build fails from tests for code that didn't exist anymore
* Refactor project structure to follow original guidlines
* Added doc comments to many methods for easier generation of documentation

### CODE REVIEW

File: trianglify/src/main/java/com/sdsmdg/kd/trianglify/utilities/patterns/Rectangle.java [hunk 1]
```diff
@@ -37,6 +37,11 @@ public Rectangle(int bleedX, int bleedY, int height, int width, int cellSize, in
         grid = new ArrayList<>();
     }
 
+    /**
+     * Generates array of points arranged in a grid of rectangles with deviation from their positions
```
Comment (dakshitagrawal) [inline trianglify/src/main/java/com/sdsmdg/kd/trianglify/utilities/patterns/Rectangle.java:41]: The method generates an array of points arranged in a grid of rectangles of given size. Also the deviation in based on the variance.

File: trianglify/src/main/java/com/sdsmdg/kd/trianglify/utilities/ThreadLocalRandom.java [hunk 2]
```diff
@@ -17,9 +16,15 @@ public ThreadLocalRandom(){
 
     }
 
-    public int nextInt(int bits) {
+    /**
+     * Generates a psuedoRandom integer that have ccomputational costs of several instructions less
+     * then that of java.util.random.nextInt().
+     * @param mod limit for generation of pseudo random number
```
Comment (dakshitagrawal) [inline trianglify/src/main/java/com/sdsmdg/kd/trianglify/utilities/ThreadLocalRandom.java]: Limit as a parameter would be better than mod.

File: trianglify/src/main/java/com/sdsmdg/kd/trianglify/utilities/ThreadLocalRandom.java [hunk 3]
```diff
@@ -17,9 +16,15 @@ public ThreadLocalRandom(){
 
     }
 
-    public int nextInt(int bits) {
+    /**
+     * Generates a psuedoRandom integer that have ccomputational costs of several instructions less
```
Comment (dakshitagrawal) [inline trianglify/src/main/java/com/sdsmdg/kd/trianglify/utilities/ThreadLocalRandom.java]: has computational costs of several instructions less than that of ...

**Comment by dakshitagrawal** [comment]
I have had a look at the documentation, not at the code.

File: trianglify/src/main/java/com/sdsmdg/kd/trianglify/utilities/patterns/Rectangle.java (hunk 1 above)
Comment (suyashmahar) [reply trianglify/src/main/java/com/sdsmdg/kd/trianglify/utilities/patterns/Rectangle.java:41]: I'll correct this :smiley:

File: trianglify/src/main/java/com/sdsmdg/kd/trianglify/utilities/ThreadLocalRandom.java (hunk 2 above)
Comment (suyashmahar) [reply trianglify/src/main/java/com/sdsmdg/kd/trianglify/utilities/ThreadLocalRandom.java]: Limit is same as mod. Returned value is `generatedValue % limit` This limits the range of output.

File: trianglify/src/main/java/com/sdsmdg/kd/trianglify/utilities/ThreadLocalRandom.java (hunk 3 above)
Comment (suyashmahar) [reply trianglify/src/main/java/com/sdsmdg/kd/trianglify/utilities/ThreadLocalRandom.java]: I'll fix this :+1:

**Comment by dakshitagrawal** [comment]
We should give a 'how to use' heading. Inside it will show an example of importing the library. We can give a list of the various color palettes as well.

**Pull request merged**